### ISSUE TITLE
Cleanup File URLs

### ISSUE THREAD

**Issue by piraka9011** [Tarteel-io/tarteel-api#215]
Some of the file URLs of the `AnnotatedRecording` Model have some extra query strings which are unnecessary.
ex. 'https://tarteel-data.s3.amazonaws.com/media/2_189_145002153_dSs5DRg.wav?AWSAccessKeyId=...&Signature=...&Expires=...'

The string should just be: https://tarteel-data.s3.amazonaws.com/media/2_189_145002153_dSs5DRg.wav'

**Comment by shafi-sohail** [comment]
@piraka9011 Hey, 
I want to take up this issue.   However, I'm new to opensource contribution and have a few questions.

Will I need to setup the whole environment for this one issue?
I am not able to locate the required file URLs.
If I understood correctly the steps are first to fork repo, make changes, submit pull request.

**Comment by piraka9011** [comment]
@shafi-sohail welcome to open source :) 
I recommend you take a look at [Django](https://docs.djangoproject.com/en/2.2/intro/) and familiarize yourself with it as well as [regex](https://www.tutorialspoint.com/python/python_reg_expressions.htm).
Yes, you will have to setup your environment to test is out completely, however, I recommend you just create a simple proof of concept (PoC) python file in the utilities folder for example that takes in a string that looks like the first example and outputs a cleaned up one.

**Comment by shafi-sohail** [comment]
@piraka9011 

>create a simple proof of concept (PoC) python file
 
What should I do after? Should I just add that file in my forked repo and submit pull request?

BTW is the issue in the file https://d2sf46268wowyo.cloudfront.net/datasets/tarteel_v1.0.csv 
in `Tarteel-ML/download.py`?

**Comment by piraka9011** [comment]
@shafi-sohail Yes, add the file in the forked repo, create a PR, and then we can review the changes before merging.

I don't think the file has to do anything with it, rather it's with the naming of the recordings in the DB.
